app/src/main/java/ca/chronofit/chrono/util/adapters/LapViewAdapter.kt

```diff
@@ -18,10 +18,16 @@ class LapViewAdapter(private val data: List<LapObject>) :
         private val lapTime: MaterialTextView = itemView.findViewById(R.id.lap_time)
         private val totalTime: MaterialTextView = itemView.findViewById(R.id.total_time)
 
-        fun bind(lap: LapObject) {
-            lapNum.text = lap.lapNum.toString()
-            lapTime.text = formatTime(getTime(lap.lapTime), ".")
-            totalTime.text = formatTime(getTime(lap.totalTime),".")
+        fun bind(lap: LapObject, position: Int) {
+            if (position == 0) {
+                lapNum.text = ""
+                lapTime.text = ""
+                totalTime.text = ""
+            } else {
+                lapNum.text = lap.lapNum
+                lapTime.text = formatTime(getTime(lap.lapTime.toLong()), ".")
```

Comment:
I think the formatTime stuff should be in a try catch block. If the format isnt correct the app is going to crash. Correct me if im wrong. If you put it in a try catch block, if the formatting fails, you can just have an empty field with a logged error. That would also get rid of you needing to use this if block.

Reply:
Hmm I'd have to test it out, if it does in fact just spit out an empty field then it makes sense to do that yeah

Reply:
lmk what its behavior is. Id like to know.

Reply:
Didn't work the way it was supposed to so just sticking with the way it was